apps/condo/domains/common/utils/moneyToWords/index.js

```diff
@@ -0,0 +1,146 @@
+const LOCALES = require('./locales')
+
+const formatRegExp = /\$([a-z]+)/gi
+
+let currency, smallDischarges, bigDischarges
+
+const setWordsToConvert = (locale = 'ru', currencyCode = 'RUB') => {
+    if (!(locale in LOCALES)) {
+        throw new Error(`Unknown Locale "${locale}"`)
+    }
+    const wordsToConvert = LOCALES[locale]
+    if (!(wordsToConvert.currency[currencyCode])) {
+        throw new Error(`Unknown currencyCode "${currencyCode}"`)
+    }
+    currency = wordsToConvert.currency[currencyCode]
+    smallDischarges = wordsToConvert.smallDischarges
+    bigDischarges = wordsToConvert.bigDischarges
+}
+
+const defaultOptions = {
+    locale: 'ru',
+    currencyCode: 'RUB',
+    format: '$WholeString $wholeCurrency $decimalString $decimalCurrency',
+}
+
+const moneyToWords = (input, options = {}) => {
+    options = Object.assign({}, defaultOptions, options)
+
+    if (options.locale) setWordsToConvert(options.locale, options.currencyCode)
+
+    if (!isValidNumber(input)) throw new Error(`Invalid Number '${input}'`)
+
+    let [base = '0', dop = '00'] = parseNumber(input)
```

Comment:
If this is about decimal part of money, like 20.05 or smth like this, you should take thta info from currency

example: BTC, 0.02928294 is normal for crypto

Reply:
> example: BTC, 0.02928294 is normal for crypto

It seems that for regular currencies two decimal places should be enough. You won't be able to pay for an amount less than one kopeck, cent or any other regular currency

!Not for all currencies, there are exceptions

Reply:
I don't think we should focus on cryptocurrencies

Reply:
> If this is about decimal part of money, like 20.05 or smth like this, you should take thta info from currency

But if it can be customized, it will be good

Reply:
Perhaps we should describe some limitations so as not to complicate the utility.